sdk/eventhub/eventhubs-checkpointstore-table/src/tableCheckpointStore.ts

```diff
@@ -62,15 +215,27 @@ export class TableCheckpointStore implements CheckpointStore {
    *  - `abortSignal`: A signal used to request operation cancellation.
    *  - `tracingOptions`: Options for configuring tracing.
    */
-  async listCheckpoints(): /*
+  async listCheckpoints(
     fullyQualifiedNamespace: string,
     eventHubName: string,
-    consumerGroup: string,
-    options: OperationOptions = {}
-    */
-  Promise<Checkpoint[]> {
-    console.log("no checkpoints to list");
-    throw new Error("not implemented");
+    consumerGroup: string
+  ): Promise<Checkpoint[]> {
+    const partitionKey = `${fullyQualifiedNamespace} ${eventHubName} ${consumerGroup} Checkpoint`;
+    const checkpoints: Checkpoint[] = [];
+    const entitiesIter = this._tableClient.listEntities<CheckpointEntity>({
+      queryOptions: { filter: odata`PartitionKey eq ${partitionKey}` }
+    });
+    for await (const entity of entitiesIter) {
+      checkpoints.push({
+        consumerGroup,
+        eventHubName,
+        fullyQualifiedNamespace,
+        partitionId: entity.rowKey,
+        offset: parseInt(entity.offset),
+        sequenceNumber: parseInt(entity.sequencenumber)
```

Comment:
When calling parseInt, we should also specify the base that our number is in (base 10 in this case).
```suggestion
        offset: parseInt(entity.offset, 10),
        sequenceNumber: parseInt(entity.sequencenumber, 10)
```